src/signals_notebook/entities/tables/cell.py

```diff
@@ -95,7 +95,7 @@ class Config:
 class CellContent(GenericModel, Generic[CellContentType]):
     value: CellContentType
     values: Optional[List[CellContentType]] = None
-    type: Optional[EntityType] = None
+    type: Optional[Union[EntityType, ObjectType]] = None
```

Comment:
Andrey's bugfix will be merged.

Reply:
type: Optional[Union[EntityType, MaterialType]] = None